### PR TITLE
NXP-30864: Add git credentials to ci

### PR DESCRIPTION
**Pull request by tfowley**

### CODE REVIEW

**Review by kevinleturc — DISMISSED** [review]

**Review by ataillefer — COMMENTED** [review]
Change is good but I would reword the commit message.
We try to explain _what_ we do or _why_ we do it, not _how_. The how is in the changeset itself.
This makes the Git history readable and comprehensive by anyone.
So, here, we could have: "Fix Git push pipeline step"

**Review by ataillefer — APPROVED** [review]

**Review by kevinleturc — APPROVED** [review]
